src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -41,10 +56,20 @@ public function configureOptions(OptionsResolver $resolver)
                     return null;
                 }
 
-                return $this;
+                $regions = $options['regions'];
+
+                return new CallbackChoiceLoader(function () use ($regions) {
+                    return self::getTimezones($regions);
+                });
             },
             'choice_translation_domain' => false,
+            'input' => 'string',
```

Comment:
Something else is troubling me. With this new implementation can we still use this choice field as multiple? I guess we should handle an array of timezones too to be fully BC.